### ISSUE TITLE
spiffs_create_partition_image not working after update (IDFGH-1696)

### ISSUE THREAD

**Issue by Mair** [espressif/esp-idf#3936]
I updated my ESP-IDF today and a project that was working is now no longer working,
When trying to write spiffs at flash time using the spiffs_create_partition_image in the CMakelists.txt
I am now getting the following error

CMake Error at main/CMakeLists.txt:9 (spiffs_create_partition_image):
  Unknown CMake command "spiffs_create_partition_image".

CMakeLists.txt as follows

```
set(COMPONENT_SRCS "main.c"
                    "server.c"
                    "connect.c"
                    "LM74A.c")
set(COMPONENT_ADD_INCLUDEDIRS ".")

register_component()

spiffs_create_partition_image(spiffs ../html FLASH_IN_PROJECT)
```

**Comment by renzbagaporo** [comment]
Hi. Can you give more information about the IDF version you've updated to, as well as the full build logs. Also, can you verify that you are able to build `$IDF_PATH/examples/storage/spiffsgen` example?

**Comment by Mair** [comment]
@renzbagaporo 
[CMakeOutput.log](https://github.com/espressif/esp-idf/files/3514081/CMakeOutput.log)
[CMakeError.log](https://github.com/espressif/esp-idf/files/3514082/CMakeError.log)


idf version is v3.2.2-dirty
I downloaded a fresh copy of the windows tools installer yesterday so I'm not sure what the exensia tool version is.
The spiffsgen example has disapeared out of the repo https://github.com/espressif/esp-idf/tree/39f090a4f/examples/examples/storage/spiffsgen gives a 404. 
and naturally it has disappeared for me to


 output looks as follows

```
-- Adding linker script C:/Users/mairs/esp/esp-idf/components/esp32/ld/esp32.rom.spiram_incompatible_fns.ld
-- Building empty aws_iot component due to configuration
CMake Error at main/CMakeLists.txt:9 (spiffs_create_partition_image):
  Unknown CMake command "spiffs_create_partition_image".


-- Configuring incomplete, errors occurred!
See also "C:/_esp32/esp32-course/_13_internet/_9_xx_Server_6_sta_tmp/build/CMakeFiles/CMakeOutput.log".
See also "C:/_esp32/esp32-course/_13_internet/_9_xx_Server_6_sta_tmp/build/CMakeFiles/CMakeError.log".
cmake failed with exit code 1
```

**Comment by renzbagaporo** [comment]
Ah, so you downgraded versions @Mair? `spiffsgen` is not available on v3.2, as it was introduced in v4.0.

The right link is https://github.com/espressif/esp-idf/tree/39f090a4f/examples/storage/spiffsgen. (`examples` was repeated in your link).

**Comment by Mair** [comment]
@renzbagaporo 
Uhh, that's interesting. 

OK let me upgrade and see how it goes
